src/helm/benchmark/scenarios/scenario.py

```diff
@@ -148,6 +148,9 @@ class Instance:
     contrast_references: Optional[List[List[Reference]]] = None
     """References for the perturbed input above (if available)"""
 
+    original_instance: Optional["Instance"] = None
```

Comment:
This means that original instances are stored twice - once in the outermost instances list, and once as a nested `original_instance` in the perturbed instance.

Is there a design that does not do this double storing? For instance, in the metrics, you can access the original instances by looking for the corresponding instance with the same ID and without a perturbation description.

Reply:
@yifanmai, What do you mean by the outermost instances list? I don't think we store the original instances unless `should_include_original_eval` is set to True and that value is not always True (see this comment https://github.com/stanford-crfm/helm/pull/1939#discussion_r1392160329).